spec.md

```diff
@@ -2422,6 +2465,18 @@ SInt("h-2A")
 SInt("h-2a")
 ```
 
+## Enum Expressions
+
+An enumeration can be constructed by applying an enumeration type to a variant
+tag and a data value expression. The data value expression may be omitted when
+the data type is `UInt<0>(0)`, where it is implicitly defined to be
```

Comment:
```suggestion
the data type is `UInt<0>`{.firrtl}, where it is implicitly defined to be
```

The type is `UInt<0>`, the value is `UInt<0>(0)`.  Added the `{.firrtl}` while visiting.